fixtures/v2/api/getDomainTransfer/success.http

```diff
@@ -0,0 +1,29 @@
+HTTP/1.1 200 OK
+Server: nginx
+Date: Mon, 27 Apr 2020 19:40:00 GMT
+Content-Type: application/json; charset=utf-8
+Transfer-Encoding: chunked
+Connection: keep-alive
+Status: 200 OK
+X-RateLimit-Limit: 2400
+X-RateLimit-Remaining: 2398
+X-RateLimit-Reset: 1588019949
+ETag: W/"8404bf2739e44e9f69f3a5199466776d"
+Cache-Control: max-age=0, private, must-revalidate
+X-Request-Id: 4818d867-1f72-4619-ab46-d345e6dd25eb
+X-Runtime: 0.092854
+Strict-Transport-Security: max-age=31536000
+
+{
+  "data": {
+    "id": 1,
+    "domain_id": 1,
+    "registrant_id": 1,
+    "state": "cancelled",

```

Comment:
It turns out the transfer object doesn't have this state documented.

```
    DomainTransfer:
      type: object
      properties:
        id:
          type: integer
        domain_id:
          type: integer
        registrant_id:
          type: integer
        state:
          type: string
          enum:
            - new
            - transferring
            - transferred
            - failed
```

Can you please update it? It's also possible the DomainRegistration object is affected.